io/EptReader.hpp

```diff
@@ -71,6 +74,14 @@ class PDAL_DLL EptReader : public Reader
     std::string getName() const override;
 
 private:
+    uint64_t m_nodeId = 1;
+    std::unique_ptr<PointTable> m_pointTable;
```

Comment:
this point table and view act as buffers, could you please rename to m_bufferPointTable and m_bufferPointView?